plutus-core/index-envs/src/Data/RandomAccessList/RelativizedMap.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE TypeFamilies         #-}
+{-# LANGUAGE UndecidableInstances #-}
+module Data.RandomAccessList.RelativizedMap (RelativizedMap (..))where
```

Comment:
maybe don't expose the constructor?

Reply:
It's basically just for testing so I think it's okay!